### PR TITLE
fix(cron): added scheduled tasks limit details

### PR DESCRIPTION
**Pull request by yohann-bacha**

### CODE REVIEW

File: src/_posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md [hunk 1]
```diff
@@ -142,3 +142,8 @@ Of course, it works in the same way as the parent app.
 ### Where can we see logs of the tasks executed?
 
 Tasks logs are included in the [application logs]({% post_url platform/app/2000-01-01-logs %}) next to other containers logs.
+
+### How many Scheduled tasks can I create ?
+
+For an application, you can create up to 5 scheduled tasks. For an increased limit, contact the support
```
Comment (brandon-welsch) [inline src/_posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md:148]: I think it's better like that.

> For an increased limit, contact the support on the embedded chat or at [support@scalingo.com](mailto:support@scalingo.com).

If you need more scheduled tasks, contact the support on the embedded chat or at [support@scalingo.com](mailto:support@scalingo.com) to increase this limit.

File: src/_posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md [hunk 2]
```diff
@@ -142,3 +142,8 @@ Of course, it works in the same way as the parent app.
 ### Where can we see logs of the tasks executed?
 
 Tasks logs are included in the [application logs]({% post_url platform/app/2000-01-01-logs %}) next to other containers logs.
+
```
Comment (brandon-welsch) [inline src/_posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md:145]: Can you update `modified_at` at this beginning of the page?

**Comment by btrd** [comment]
@yohann-bacha @brandon-welsch Can this PR be merged?

**Comment by brandon-welsch** [comment]
For me it's okay yes. Just one thing before the merge, can we change the timestamp? 😃

File: src/_posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Scalingo Scheduler - Run Scheduled Tasks
 nav: Scalingo Scheduler
-modified_at: 2022-10-24 10:00:00
+modified_at: 2023-01-27 14:19:00
```
Comment (btrd) [inline src/_posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md:4]:
```suggestion
modified_at: 2023-02-22 00:00:00
```

Comment (yohann-bacha) [reply src/_posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md]: Done

**Review by brandon-welsch — APPROVED** [review]
LGTM

**Pull request merged**